extension/contents/page-observer.ts

```
@@ -319,24 +345,122 @@ class PageObserver {
+          case 'i':
+            markdown += '*' + elem.textContent?.trim() + '*';
+            break;
+            
+          case 'code':
+            markdown += '`' + elem.textContent?.trim() + '`';
+            break;
+            
+          case 'pre':
+            markdown += '\n```\n' + elem.textContent?.trim() + '\n```\n\n';
+            break;
+            
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)